components/Search/Searchbar.tsx

```diff
@@ -110,6 +131,36 @@ export default function Searchbar() {
                }).filter(x => !!x).slice(0, 5)}
             </>
             : null}
+         {titles && searchValue.length > 0 ?
+            <>
+               {titles.map((title) => {
+                  if (!title.name.toLowerCase().startsWith(searchValue)) {
+                     return null;
+                  }
+
+                  if (title.challengeId === undefined) { // TODO titles with no challange
+                     return;
+                     return <TitleCard
+                     key={title.id}
+                     title={title.name}
+                     url={"/challenges/" + title.challengeId} // TODO think of a link
+                     image={contentService.getChallengeTokenIcon(parseInt("1"), "MASTER")} // TODO think of a icon
+                  />;
+                  }
+
+                  if (title.type === "CHALLENGE") {
```

Comment:
type can be either `CHALLENGE` or `DEFAULT`

Reply:
Fixed